src/components/Clients/ClientListTableRow.jsx

```diff
@@ -46,13 +49,7 @@ const ClientListTableRow = ({ labelId, client, state, dispatch }) => {
 
   // Event handler for deleting client from client list
   const handleDeleteClient = async () => {
-    if (
-      !window.confirm(
-        `You're about to delete ${client.person} from your client list, do you wish to continue?`
-      )
-    ) {
-      return;
-    }
+    setShowClientDeleteModal(false);
```

Comment:
Not really an issue per-se since I believe this is appropriate too. But the modal seems to close even without this line when I click on "Delete Client".

https://github.com/codeforpdx/PASS/assets/14917816/f6839acb-401f-4fc0-8526-9d2e0f84ab10

I wonder if it has something to do with the client being removed from the list so the modal disappears on its own.

Reply:
It closes when the action is complete.  I was having it close early, so user could see the status notifications that are displayed on the table.

Could move the notifications to the modal, as Tim said below.  

Or have notifications some other way:
 - Just a spinner while it's running (https://mui.com/material-ui/react-progress/#circular) and then a snackbar for error or success (https://mui.com/material-ui/react-snackbar/#customization)
 - Interactive integration (https://mui.com/material-ui/react-progress/#interactive-integration)

Reply:
@timbot1789 since you mentioned it in another comment, take a look at these options as well and see what you think.

Reply:
I like the snackbar because we'll probably want to use something similar for other messages throughout the system.

Reply:
This is related to issue #227 and it looks like Russell just assigned himself to it